src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -5,35 +5,34 @@
 #include <llprint.h>
 
 #define UNDEF_SYMBS 64
-
-/* Assembly function for sinv from new component */
-extern void *__inv_test_entry(int a, int b, int c);
+#define CMP_UTYPMEM_SZ (1 << 26)
+#define THD_PRIO 5
+#define THD_PERIOD 10
```

Comment:
Because the components with schedulers have their own tcaps, the parent (llbooter) delegates time to these child schedulers. This is why sl in llbooter needs period & budget for periodic replenishments. 

I think we need to namespace these correctly. Like `CHILDSCHED_THD_XXXXX` or something.